start.sh

```diff
@@ -84,7 +85,37 @@ while [ "$(curl -k -s -o /dev/null -w ''%{http_code}'' -u "$ADMIN_USER:$ADMIN_PA
 done
 
 #
-# Apply the code example's configuration via a RESTCONF PATCH
+# For the HAAPI example, update configuration dynamically
+#
+cd ../$EXAMPLE_NAME
+if [ "$EXAMPLE_NAME" == 'haapi' ]; then
+
+  if [ "$ANDROID_FINGERPRINT" == '' ]; then
+    ANDROID_FINGERPRINT='67:60:CA:11:93:B6:5D:61:56:42:70:29:A1:10:B3:86:A8:48:C7:33:83:7B:B0:54:B0:0A:E3:E1:4A:7D:A0:A4'
+  fi
+  if [ "$ANDROID_SIGNATURE_DIGEST" == '' ]; then
+    ANDROID_SIGNATURE_DIGEST='Z2DKEZO2XWFWQnApoRCzhqhIxzODe7BUsArj4Up9oKQ='
+  fi
+  if [ "$APPLE_APP_ID" == '' ]; then
+    APPLE_APP_ID='io.curity.cat.ios.client'
```

Comment:
The app ID in the UI demo is actually now `io.curity.haapidemo`, but scripts in that app set the env variable anyway. Not sure if we need the defaults here. Maybe it's better to fail when these values are not set? Not sure...

Reply:
Yeah - io.curity.cat.ios.client is used by the model SDK example so I fall back to that here. I want the newer one to be better named, without impacting the model SDK example.

Reply:
That makes sense. I forgot about the model SDK demo.